higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -313,8 +313,10 @@ <h3 class="heading-subtitle heading-subtitle--edu-label">Confirmation</h3>
 
       //Pre-existing conditions Yes & cleanup
       let preexistingRadioKey = sessionStorage.getItem('preexistingRadio');
-      if(preexistingRadioKey === 'true' || preexistingRadioKey === 'not_sure') {
-        preexistingconditionsYes.checked = true;
+      if(preexistingRadioKey === 'yes' || preexistingRadioKey === 'not_sure') {
+        if(preexistingRadioKey === 'yes')preexistingconditionsYes.checked = true;
+        else preexistingconditionsNotsure.checked = true;
```

Comment:
I don't think that this is valid javascript if/else statement

Reply:
@rudigiesler updated